src/runtime_src/core/common/config_reader.h

```diff
@@ -711,6 +711,13 @@ get_force_program_xclbin()
   return value;
 }
 
+inline std::string
+get_dummy_app_context_type()
```

Comment:
can we remove "dummy" from the name and instead just call it "context_type"

Reply:
Should I rename this to "hardware_context_type" ? @stsoe mentioned the parameter name should be more descriptive.

Reply:
> Should I rename this to "hardware_context_type" ? @stsoe mentioned the parameter name should be more descriptive.

Yes, that would be better.